.github/workflows/pr-title-labeler.yml

```diff
@@ -1,9 +1,12 @@
 name: check PR title and label appropriately
 on:
+  push:
+    branches:
+      - main
   pull_request:
-    types: [opened, edited, synchronize]
+    types: [opened, edited, synchronize, auto_merge_enabled, reopened]
     branches:
-      - main, staging
+      - main
```

Comment:
this is not quite right (see other PR)

Reply:
Is that necessary given that MANAGE doesn't have a staging branch?

Reply:
.... touche!